pom.xml

```diff
@@ -66,4 +66,33 @@
             <systemPath>${project.basedir}/OnlineServices-1.0-SNAPSHOT.jar</systemPath>
         </dependency>
     </dependencies>
+    <build>
```

Comment:
OnlineService is still not in the GeneralService jar, after running mvn clean install open the GeneralService jar using 7zip to see what is in there